semanticdb/semanticdb3/semanticdb3.md

```diff
@@ -1642,6 +1650,8 @@ Notes:
 * Supported properties for parameter symbols are:
   * `IMPLICIT`: set for `implicit` parameters, as well as desugared context
     bounds and view bounds (see above).
+  * `VALPARAM`: set for `val` parameters of primary constructors.
```

Comment:
They are different. This will documented as part of a larger effort to comprehensively cover synthetics: https://github.com/scalameta/scalameta/issues/1329.